CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Methods `next_up()` and `next_down()` for floating-point,
+  based on the IEEE-754 standard.
```

Comment:
What is based on the IEEE-754 standard? Can next be defined in another way? Does this only apply for IEEE bias?

Reply:
It was meant to convey that the behaviour of the methods follows the IEEE-754 standard. I don't know if there's any other definition.

I do see that the meaning of "based" here is unclear and should be changed. Perhaps the standard should not be mentioned at all here? Describing the behaviour in the changelog is superfluous.